api-tests/apply_route_test.go

```diff
@@ -529,6 +568,44 @@ func TestApplyPGReturnedVersions(t *testing.T) {
 	}
 }
 
+func TestApplyPSReturnedVersions(t *testing.T) {
+	cli := cli()
+
+	cases := []struct {
+		apply    string
+		operator string
+		version  string
+	}{
+		// test latest
+		{"latest", "0.2.0", "8.0.28-20"},
+
+		// test recommended
+		{"recommended", "0.2.0", "8.0.28-20"},
+
+		// test exact
+		{"8.0.27", "0.2.0", "8.0.27-18"},
+
+		//test with suffix
+		{"8-latest", "0.2.0", "8.0.28-20"},
```

Comment:
In PXC we have `8.0-recommended` so I propose we have the same for consistency reasons and also we don't know what Oracle will do with MySQL version.
So let's have `8.0-recommended` and `8.0-latest`

Reply:
https://github.com/Percona-Lab/percona-version-service/pull/88/commits/6a986830e4a903b5d402b33cc3b5a5a0695648e7